app/views/accounts/_emails.html.erb

```diff
@@ -3,6 +3,9 @@
     <h3 class="h2 margin-0">
       <%= t('account.index.email_addresses') %>
     </h3>
+    <% if flash[:email_error] %>
+      <%= render AlertComponent.new(type: :error, id: 'emails', class: 'margin-bottom-1', message: flash[:email_error]) %>
```

Comment:
In [prior work](https://github.com/18F/identity-idp/pull/5779), the ID was added to be able to focus the content when the page is loaded by adding the hash fragment to the URL (e.g. `/account#phones`). Do we want to do the same thing with this by including it in the redirect URL? Otherwise, we can probably leave out the `id`.

Reply:
I decided to include it in the redirect url to ensure the user sees it and to keep things consistent with other button error redirects.